examples/word-count/tests/test_word_count.py

```diff
@@ -8,8 +8,8 @@
 path = os.path.join(current_dir, "zen-of-python.txt")
 
 
-@pytest.fixture(scope="session", autouse=True)
-def textfile():
+@pytest.fixture(scope="session")
+def contents() -> str:
```

Comment:
Does it work with Python 3.5?

Reply:
Yes https://docs.python.org/3.5/library/typing.html